src/mmword.c

```diff
@@ -285,9 +284,9 @@ l7220:  i=i+m-1;
                  addTag_, 0);
              fprintf(f3_fp_, "%s\n", tmpLine);
            }
+           // Output original deleted lines // freeTempAlloc();
+           // Clear vstring stack
```

Comment:
This comment got mangled, the freeTempAlloc() should go after the printf, and the "clear vstring" comment applies to it

Reply:
Done